financial-connections/src/main/java/com/stripe/android/financialconnections/domain/SaveAccountToLink.kt

```diff
@@ -90,9 +100,13 @@ internal class SaveAccountToLink @Inject constructor(
         }.mapCatching {
             action(selectedAccountIds)
         }.onSuccess { manifest ->
-            storeSavedToLinkMessage(manifest, selectedAccountIds.size)
+            if (!isRelink) {
+                storeSavedToLinkMessage(manifest, selectedAccountIds.size)
+            }
```

Comment:
We don’t want to show `[…] and was saved to Link`, since the account is technically already in Link.